### PR TITLE
feat: translate module

### LINKED ISSUE #21: Add new module : translator
https://github.com/the-vision/jarvis-telegram/wiki

### PR DESCRIPTION
**Pull request by adityabisoi**
Closes #21 

Description:
* Since telegram doesn't provide a  to save the state of a user and since the project doesn't use any database, a workaround is to show the destination language, and then providing a URL to Google Translate with pre-filled and translated data.
Screenshot from 2020-03-10 09-15-03

### CODE REVIEW

**Review by tanisha03 — APPROVED** [review]

**Comment by swapagarwal** [comment]
https://github.com/the-vision/jarvis-core/issues/25 needs to be resolved first.

**Comment by adityabisoi** [comment]
@swapagarwal @tanisha03 @TeAmP0is0N Is text tripping of `intents` is possible, as in this PR? I have stripped the text to extract the inputs for the intent. Stripping `message.text` works on my development, but is this implementable in the production bot?

File: modules.py [hunk 1]
```diff
@@ -43,6 +43,14 @@ def reply(bot, message, intent, entities):
         result = random.choice(['Heads', 'Tails'])
         bot.send_photo(message.chat.id, photo=coin_images[result],
                        reply_to_message_id=message.message_id)
+    elif intent[:9] == 'translate':
+        word = message.text[9:]
```
Comment (swapagarwal) [inline modules.py]: This would work, but it's not advisable. Ideally, you'd define an entity containing the text to be translated.

**Comment by adityabisoi** [comment]
@swapagarwal @tanisha03 @TeAmP0is0N I have made the necessary changes. Please review.

**Review by RohitRathore1 — APPROVED** [review]

**Pull request merged**